.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,21 @@
+name: Dependabot auto-merge
+on: pull_request
+
+permissions:
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Approve a PR
+        run: gh pr review --approve "$PR_URL"
+        env:
+          PR_URL: ${{github.event.pull_request.html_url}}
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2
+        with:
+          target: minor
```

Comment:
@nmerget I could think about to start with `patch` even only here, as `minor` might imply to some changes that would improve our software, and that we should that for be aware of and have a closer look at minor releases / changelogs. @dkolba what do you think ?

Reply:
@mfranzke have you ever looked closer at a minor change for a dependency? 

I think we should start with this auto-merge for patch and minor and see if it fits :)

Reply:
@nmerget, yes, every time. That‘s why I‘m asking.

Reply:
@dkolba what‘s your opinon? In the end its a tradeoff in between making informed decisions / identifying rework early by being able to read through a changelog for minor updates vs. more automation.

Reply:
I changed it to patch and interval to weekly. So we can check every monday for minor and major changes